app/controllers/keywords_controller.rb

```diff
@@ -10,6 +10,21 @@ def index
            })
   end
 
+  def show
+    keyword = Keyword.find(params['id'])
+
+    search_entries_query = KeywordSearchEntriesQuery.new.with_keyword(keyword.id)
+
+    render(locals: {
+             keyword: keyword,
```

Comment:
how about just passing the `search_entries_query` @Goose97 
```suggestion
             search_entries_query: search_entries_query
```

Reply:
Yeah, that's what I've been wondering too. One one hand, it does make the code less verbose. But it also add more responsibility for the view, namely querying the database.

In my mind, views should be pure and dumb. It only receives inputs as plain data and outputs a HTML / JSON payload. That might be over thinking for this case though 😄

Reply:
The view would not do the querying as this is handle by the query object. The controller invokes the query objects and then pass an object to the view (similar to `assigns` in Phoenix) so I do not see any concerns here :-)

Reply:
Fair point 👍